docs/releases/4.2.txt

```diff
@@ -306,6 +306,10 @@ Models
   longer required to be set on PostgreSQL, which supports unlimited ``VARCHAR``
   columns.
 
+* :attr:`Field.choices <django.db.models.Field.choices>` now also accepts
```

Comment:
https://github.com/django/django/blob/c914d6cff176ae6bfab2f33a84bcfd45208f1894/docs/ref/models/fields.txt#L82-L93

would require a fresh paint :)

With a little `.. versionadded:: X.Y` section

Reply:
I updated the docs to the best of my knowledge. I used `4.2` as a placeholder, but as you pointed out, it seems too late for it now. Is it the "so far away" `5.0` then?